src/Unicorn.Tests/Predicates/SerializationPresetPredicateTests.cs

```diff
@@ -37,8 +38,13 @@ public void ctor_ThrowsArgumentNullException_WhenNodeIsNull()
 		[InlineData("/implicit-nochildren/ignoredchild", false)]
 		[InlineData("/implicit-nochildren/ignored/stillignored", false)]
 		[InlineData("/implicit-nochildrenwithextrachars", false)]
-		// SOME-CHILDREN test config
-		[InlineData("/somechildren", true)]
+        // TEMPLATE-ID test config
```

Comment:
Check your whitespace; it looks like tabs have converted to spaces in most of the code.

There is an editorconfig file that defines the project style; VS2015 has a plugin that can follow this; VS2017 follows it natively.